pom.xml

```diff
@@ -2001,15 +2001,26 @@
                 <param>com.puppycrawl.tools.checkstyle.DetailNodeTreeStringPrinter</param>
                 <param>com.puppycrawl.tools.checkstyle.AstTreeStringPrinter</param>
                 <param>com.puppycrawl.tools.checkstyle.TreeWalker</param>
-                <param>com.puppycrawl.tools.checkstyle.Main</param>
               </targetClasses>
               <targetTests>
                 <param>com.puppycrawl.tools.checkstyle.DetailNodeTreeStringPrinterTest</param>
                 <param>com.puppycrawl.tools.checkstyle.AstTreeStringPrinterTest</param>
                 <param>com.puppycrawl.tools.checkstyle.TreeWalkerTest</param>
-                <param>com.puppycrawl.tools.checkstyle.MainTest</param>
+                <param>com.puppycrawl.tools.checkstyle.checks.blocks.*</param>
```

Comment:
@rnveach the thing is the majority of test are written in a functional way. And during execution many of them work with a TreeWalker. Adding check tests to this profile increases mutation coverage on ±6%.